api/proto/teleport/trust/v1/trust_service.proto

```diff
@@ -70,3 +74,27 @@ message UpsertCertAuthorityRequest {
   // The certificate authority to create or update.
   types.CertAuthorityV2 cert_authority = 1;
 }
+
+// GenerateHostCertRequest is the request for GenerateHostCert.
+message GenerateHostCertRequest {
+  // key is the SSH public key that the certificate should include.
+  bytes key = 1;
+  // host_id is the ID of the host the certificate should include.
```

Comment:
curious, what is the source of `host_id`? do we generate a uuid or that is picked up from the host itself?

Reply:
This is from the host requesting the certificate - it's generally a UUID that's related to that specific agent etc. It's generated on the first start of the node.